.mk-lib/README.md

```diff
@@ -0,0 +1,95 @@
+# Docker compose makefile
```

Comment:
I didn't know about this package, I'll check it later, but it adds a lot of files I don't know what is the added value

Reply:
It speeds up targets generation, provides auto help and more enhancements.

Btw, this is not mandatory :)